part2/erc20-js.md

```diff
@@ -9,7 +9,7 @@ For now, qtumjs relies on `qtumd`'s RPC service, so make sure that the docker co
 ```
 docker run -it --rm \
   --name myapp \
-  -v `pwd`/dapp \
+  -v $PWD:/dapp \
```

Comment:
I believe both `` `pwd` `` and `$PWD` are valid here. Was there a specific reason you made this change?

Reply:
Never mind, it seems like `$PWD` is necessary here to for compatibility with [fish](https://fishshell.com). I think the changes are good.